### DISCUSSION TITLE
Vimar 14594.0 slat orientation

### DISCUSSION THREAD

**Discussion by sginestrini** [Koenkk/zigbee2mqtt#20180] [Devices]
Hi, I'm looking to get a few for controlling roller shutter with slat orientation. The things I noticed is that cover (tilt) is not exposed in z2m, am I right? From the manual https://faidate.vimar.com/irj/go/km/docs/z_catalogo/DOCUMENT/ZIS_30804-20594EN_FI.106648.pdf I can see that it can be set and calibrated, but can I do it from zigbee? 
Calling Vimar didn't help, because they replied that it's Alexa's fault (even if I'm telling them that I'm not going to use Alexa😆) and everything works with their Bluetooth gateway.
Thank you

**Comment by paolo-tasca** [comment]
I'm interested in this feature too

**Reply by sginestrini** [reply]
Just buy another device as Vimar suggested me 😅
